types/video.js/video.js-tests.ts

```diff
@@ -229,6 +229,7 @@ function testComponents(player: videojs.Player) {
     myWindow.open();
     myWindow.close();
     myWindow.myFunction();
+    myWindow.handleKeyDown();
```

Comment:
IMO the test is wrong, it tests 'modal' 'handleKeyDown' which seems to be incorrect, not accepting any param. The source:
https://github.com/videojs/video.js/blob/054329899be5acbdcd579d856cf21ea27ead0c7b/src/js/modal-dialog.js#L450